Makefile

```diff
@@ -169,7 +152,7 @@ else
 endif
 
 .PHONY: cluster
-cluster: $(CTLPTL) $(KUBECTL) ## Creates kind-dev Cluster 
+cluster: get-dependencies $(CTLPTL) ## Creates kind-dev Cluster 
 	./hack/kind-dev.sh
```

Comment:
I find that we have inconsistencies in the k8s version for kind, can you bump the version inside this script? `K8S_VERSION=v1.27.3`